kanalony-storage-api/src/main/scala/ConnectorFactory.scala

```diff
@@ -0,0 +1,10 @@
+package kanalony.storage.api
+
+import com.websudos.phantom.connectors.{ContactPoint, KeySpaceDef}
+
+/**
+ * Created by elad.benedict on 2/7/2016.
+ */
+object ConnectorFactory extends IConnectorFactory {
+  override val connector : KeySpaceDef = ContactPoint.local.keySpace("my_keyspace")
```

Comment:
While this is true, it (currently) will not be reflected in code - once we have additional keySpaces, we'll simply modify the ConnectorFactory and DbClientFactory accordingly